### ISSUE TITLE
Proposal to move one of our three meeting times

### ISSUE THREAD

**Issue by Trott** [nodejs/CTC#107]
TL;DR: Proposing moving our UTC 5AM to UTC 11AM.

Updates to the meeting times spreadsheet (email me if you need the link) indicate that 5AM UTC is not a viable time anymore. Fully half of the CTC rates that time a 0 (impossible) or a 1 (less than 50% likelihood of attendance).

I've done the usual processing of the spreadsheet data. As always, I'm open to other suggestions. But I'd propose that we move the 5AM UTC to 11AM UTC and leave the other two times as they are for now. 

The proposed change to 11AM UTC is a substantial improvement for addaleax, bnoordhuis, fhinkel. It is significantly worse than 5AM UTC for trevnorris and me (But the other two times are all 4s and 5s for trevnorris and me, so I'm OK with that.)

As of today (in other words, with daylight savings in US and Canada, etc.), 11AM UTC is:

San Francisco: 4:00 am
New York: 7:00 am
Amsterdam: 1:00 pm
Moscow:	2:00 pm
Chennai: 4:30 pm
Tokyo: 8:00 pm
Sydney: 9:00 pm

Arguing against myself, the big downside is we're moving from an unviable time to a barely viable time. (9 of 20 CTC members rate this a 0 or a 1, which doesn't make it *that* much better than 5AM UTC.) But it's one that accommodates people in time zones that aren't in the Americas, and that's the main point, as the other two times are extremely convenient to folks in San Francisco, New York, etc.

**Comment by bnoordhuis** [comment]
Hurray, +1!  Would be the first CTC meeting (or, come to think of it, any node.js meeting) during regular office hours for me.

**Comment by jasnell** [comment]
(James looks at the coffee pot and wonders if 12 cups is enough to get him through a 4am CTC meeting... He sighs deeply, thinking of Ben and Anna and Rod... Ah well, he mutters.)

+1 :)

**Comment by bnoordhuis** [comment]
You're allowed to skip the odd meeting every now and then. :-)  I'd be okay with two or three hours later (still office hours!) but that might be less agreeable to the people from around Oceania.

**Comment by mhdawson** [comment]
slightly more likely to make the new time, so +1 from me.

**Comment by ChALkeR** [comment]
@Trott, that looks good to me, but I'm interested to see what algorithm is used here, or, more specifially, how is the utility function defined =).

**Comment by ChALkeR** [comment]
@Trott, my quick and dirty algorithm gave me `[6, 18, 20]`. I am not insisting that it's better, but could you take a look at that combination?

**Comment by Trott** [comment]
@ChALkeR:

My algorithm: https://gist.github.com/Trott/d1423a7583adf175eac79c207203b9c9 (you'll recognize the .tsv processing bit that I stole from you)

I only considered times that kept the UTC 16:00 and UTC 20:00 because I figured it would be an easier sell to just change one of the three meeting times rather than two or all three.

I ruled out UTC 06:00 because (as with 05:00) there are 10 CTC members who are likely to not make that time (scores of 0 or 1) so we are unlikely to have a quorum. Admittedly, that's a *somewhat* arbitrary cut-off. Other ideas are welcome. (Our IRC chat gave me the idea to treat this as a weighting multiplier somehow rather than as a cutoff, so that 9 absences is almost as bad as 10, rather than what I have now which is that 9 and below are fine while 10 and above are unacceptable. Problem with the weighting multiplier is that it can push you back towards "OK, only times that are good for the Americas are going to work" which is exactly what we *don't* want. Will require some experimentation on my part and/or someone more knowledgable than me to sort out.)

To be clear, I'm not arguing against these other proposals here. I'm just explaining how I arrived at the one I did.

**Comment by Trott** [comment]
For what it's worth, my algorithm gives the highest marks for fairness and utility to 12:00 / 17:00 / 22:00. Nicely equidistant (5-hour) times. But it would require us to agree to junk all of our current meeting times and move to three new ones, and I think that's likely to be a harder sell than just moving one of the three times.

**Comment by ChALkeR** [comment]
Is this solved by #112 (which landed) or are there any further actions needed?

**Comment by Trott** [comment]
> Is this solved by #112 (which landed) or are there any further actions needed?

@ChALkeR Seems resolved to me. Thanks for the reminder!
